src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java

```diff
@@ -101,6 +106,24 @@
     @Parameter
     boolean usePropertiesForVersion;
 
+    /**
+     * Whether to use dependency management dependencies.
+     */
+    @Parameter(defaultValue = "false")
+    private boolean useDependencyManagementDependencies;
+
+    /**
+     * Wheter to use dependencies.
+     */
+    @Parameter(defaultValue = "false")
+    private boolean useDependencies;
+
+    /**
+     * Wheter to use all resolved dependencies (defaults to true).
+     */
+    @Parameter(defaultValue = "true")
+    private boolean useArtifacts;
```

Comment:
Would it be possible to use name useAllResolvedDependencies ? That way it would correspond to other names

Reply:
Will be solved with !5